arm/Microsoft.KeyVault/vaults/readme.md

```diff
@@ -177,12 +177,12 @@ To use Private Endpoint the following dependencies must be deployed:
 
 ## Outputs
 
-| Output Name | Type | Description |
-| :-- | :-- | :-- |
-| `keyVaultName` | string | The name of the key vault. |
+| Output Name             | Type   | Description                                                  |
```

Comment:
Undo autoformatting. Seems you have a VSCode extension installed that does this for you. Please disable to autoformatting for that extension and roll the changes back